source/libraries/HelperFunctions.ts

```diff
@@ -1,5 +1,8 @@
 import * as binascii from "binascii";
+import * as sleep from "sleep";
 import * as EthjsAccount from "ethjs-account";
+import * as EthjsQuery from 'ethjs-query';
+import { ContractReceipt } from "contract-deployment";
```

Comment:
Unused; should be deleted.

Reply:
I'll remove these.

Reply:
Note: JUST `ContractReceipt` should be removed; not ethjs-query.

Reply:
Right, I was referring to `ContractReceipt` and `sleep` since Micah suggested returning a promise with setTimeout instead.

Reply:
Just a heads-up @adrake33 I went ahead and removed ContractReceipt as part of a larger push.  (Sorry for scooping you!)

Reply:
No worries, I just saw that.